core/src/main/resources/hudson/model/Job/buildTimeTrend_resources.js

```diff
@@ -3,7 +3,7 @@
  */
 function buildTimeTrend_displayBuilds(data) {
 	var p = document.getElementById('trend');
-	var isMasterSlaveEnabled = 'true' === p.getAttribute("data-is-master-slave-enabled");
+	var isMasterSlaveEnabled = 'true' === p.getAttribute("data-is-distributed-build-enabled");
```

Comment:
Any reason variable name isn’t updated?

Reply:
I also forgot it in the original PR, now done.